indico/MaKaC/PDFinterface/conference.py

```diff
@@ -1724,11 +1726,11 @@ def __init__(self, conf, badgeTemplate, marginTop, marginBottom, marginLeft, mar
         self.__marginRight = marginRight
         self.__marginColumns = marginColumns
         self.__marginRows = marginRows
-
         if registrantList == 'all':
-            self.__registrantList = self.__conf.getRegistrantsList(sort = True)
+            self.__registrantList = Registration.find(~Registration.is_deleted).all()
```

Comment:
Where's the filter by regform? `.with_parent(regform)` would do the job.

This is needed in both cases (so you can't simply put an ID from another registration form in the POSTed data)